plugins/flatpak/gs-flatpak.c

```diff
@@ -394,6 +395,23 @@ remote_is_eos_apps (FlatpakRemote *remote)
 		g_str_has_suffix (remote_host, "ostree.endlessm.com");
 }
 
+static gboolean
+mount_contains_uri (GMount *mount, const gchar *uri)
+{
+	g_autoptr(GFile) mount_root = NULL;
+	g_autofree gchar *mount_uri = NULL;
+
+	if (!mount)
+		return FALSE;
+
+	mount_root = g_mount_get_root (mount);
+	mount_uri = g_file_get_uri (mount_root);
+
+	if (g_str_has_prefix (uri, mount_uri))
```

Comment:
I think it would be better to operate on `GFile`s and use `g_file_has_parent()` here, otherwise you run into problems if the prefix string doesn’t end with a slash.